connector/producer.py

```diff
@@ -45,7 +46,7 @@
 @openerp.api.returns('self', lambda value: value.id)
 def create(self, vals):
     record_id = create_original(self, vals)
-    if self.pool.get('connector.installed') is not None:
+    if is_module_installed(self.env, 'connector') is not None:
```

Comment:
If that works, it would be clearer. You are passing self.env as the first argument of  https://github.com/OCA/connector/pull/74/files#diff-2700ffb2a55d3a936589a6fab4428330R31 called self, then access self.env. Again, I am wondering why this works at all.